_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Star Tree
+nav_order: 61
+has_children: false
+parent: Supported field types
+redirect_from:
+  - /opensearch/supported-field-types/star-tree/
+  - /field-types/star-tree/
+---
+# Star tree field type
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+Star Tree Index (STIX) pre-computes aggregations to accelerate the performance of aggregation queries. <br/>
+If STIX is configured as part of index mapping, it will be created and maintained in real-time as the data is ingested.<br/>
+OpenSearch will automatically use the Star Tree index to optimize aggregations if the fields queried are part of Star Tree's dimension fields and the aggregations are on Star Tree's metrics fields. No changes are required in the query syntax or the request parameters.
+
+For more information, see [Star Tree index]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index/)
+
+## Prerequisites
+
+To use Star Tree on your index, the following prerequisites needs to be satisfied:
+
+- Set the feature flag `opensearch.experimental.feature.composite_index.star_tree.enabled"` to `true`. For more information about enabling and disabling feature flags, see [Enabling experimental features]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/).
+- Set the `indices.composite_index.star_tree.enabled` setting to `true`. For instructions on how to configure OpenSearch, see [configuring settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index/#static-settings).
+- Set the `index.composite_index` index setting to `true` during index creation.
+- Enable `doc_values` : Ensure that the `doc_values` is enabled for the [dimensions](#ordered-dimensions) and [metrics](#metrics) fields used in your Star Tree mapping.
+- Star Tree index ideally should be used with indexes whose data is not updated or deleted, as updates and/or deletes are not accounted in Star Tree index.
+
+## Examples
+
+The following examples show how to use Star Tree index.
+
+### Star tree index mapping
+
+Define Star Tree mapping under new section `composite` in `mappings`. <br/>
+To compute metric aggregations for `request_size` and `latency` fields with queries on `port` and `status` fields, configure the following mappings:
+
+```json
+PUT logs
+{
+  "settings": {
+    "index.number_of_shards": 1,
+    "index.number_of_replicas": 0,
+    "index.composite_index": true
+  },
+  "mappings": {
+    "composite": {
+      "startree1": {
```

Comment:
does it need to be startree1, can it be called request_aggs?

Reply:
This is basically the star tree field name - which can be anything as long as its unique when multiple star trees are supported.

Reply:
I felt ```request_aggs``` or something similar is more meaningful, similar to the index names in SQL tables.

Reply:
sure changing it